forgettable-frontend/src/components/InputSelector/InputSelector.js

```diff
@@ -0,0 +1,64 @@
+import React, {useEffect} from 'react';
+import {useState} from 'react';
+import styles from './InputSelector.module.css';
+
+/**
+ * React input selector component used for text entry
+ *
+ * The styling of the component is based on that of the InputField component ("primary")
+ *
+ * Function Arguments:
+ * @param inputLabel refers to the label associated with the text input component.
+ * @param inputType refers to the "inputType" parameter as mentioned above.
+ * @param placeholder refers to the placeholder text in the component.
+ * @param dataType refers to the data format allowed as input.
+ * @param inputID refers to the ID for the data associated with the input.
+ * @param inputStateValue refers to the pre-filled value when the input is rendered.
+ * @param autoFocusState refers to the autoFocus attribute, parameter is a Boolean.
+ * @param requiredState refers to the required attribute, parameter is a Boolean.
+ * @param options refers to an array of the options to be displayed in the dropdown.
+ *
+ * @author Bill Song
+ */
```

Comment:
I'd prefer to also get a second opinion @nzbasic  but I feel like this is too verbose, especially. However, feel free to ignore this observation.

Reply:
For context, this was pretty much entirely lifted from the InputField component, which is very similar. Agree that pretty much all the param descriptions are unnecessary, and considering other components had far less documentation, I think it's good to have this fit status quo. fixed